reactDashboard/src/pages/Overview.jsx

```diff
@@ -89,12 +89,25 @@ const Overview = ({
       false
     );
   }, []);
+
+  const bannerPt1 = "We are currently in ";
```

Comment:
I reckon this isn't the way to go. A well defined and clear (though duplicated) component is now less readable. The fact that bannerPt2 exists to provide a highlight in the middle of the title is now opaque. If anything, I'd have had the text of both pages' banners in the banner component, and possible have that component select which one to render based on a Router match

Reply:
I think your option two is better - once we start adding more pages, having a separate component for each page/banner will start to get more difficult to update if styling requires to be changed, reusable components lend themselves to better and more consistent layouts and less coding required when new pages are added etc. 

Is this maybe something that can be refactored once Diane's page is ready to merge - assuming we are going to try and introduce a banner there too, but would be good to get this PR through - it is working as is, so we can have something on the dashboard for the meantime?

Reply:
I'm really not keen on it as it is - it separates text and text formatting into separate components. I can give you a hand refactoring it to something cleaner if you like :)